card/mod/core/set/all/collection.rb

```diff
@@ -228,10 +228,11 @@ def edit_fields
   end
 
   def normalized_edit_fields
-    edit_fields.map do |name, options|
-      options ||= Card.quick_fetch(name).name
+    edit_fields.map do |name_or_card, options|
+      next [name_or_card, options || {}] if name_or_card.is_a?(Card)
```

Comment:
ok, there is a way to avoid that. Instead of 
``` ruby
year = card.fetch trait: :year, new: { content: card.year }
voo.edit_structure = [[year, "Year"]]
```
I can use
```ruby
card.add_subfield :year, content: card.year
voo.edit_structure = [[:year,  "Year"]]
```
But conceptually it's the same. I have to pass an card object to edit_structure. I just don't do it directly. I use the cache instead to pass it.